docroot/modules/custom/va_gov_vba_facility/src/EventSubscriber/VbaFacilitySubscriber.php

```diff
@@ -0,0 +1,126 @@
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Drupal\core_event_dispatcher\Event\Entity\EntityViewAlterEvent;
+
+/**
+ * VA.gov VBA Facility Event Subscriber.
+ */
+class VbaFacilitySubscriber implements EventSubscriberInterface {
+
+  use StringTranslationTrait;
+
+  /**
+   * UserPerms Service.
+   *
+   * @var \Drupal\va_gov_user\Service\UserPermsService
+   */
+  protected $userPermsService;
+
+  /**
+   * The entity manager.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManager
+   *  The entity manager.
+   */
+  private $entityTypeManager;
+
+  /**
+   * The renderer service.
+   *
+   * @var \Drupal\Core\Render\RendererInterface
+   *  The renderer.
+   */
+  private $renderer;
+
+  /**
+   * Constructs a EntityEventSubscriber object.
+   *
+   * @param \Drupal\Core\StringTranslation\TranslationInterface $string_translation
+   *   The string translation service.
+   * @param \Drupal\va_gov_user\Service\UserPermsService $user_perms_service
+   *   The string translation service.
+   * @param \Drupal\Core\Entity\EntityTypeManager $entity_type_manager
+   *   The string translation service.
+   * @param \Drupal\Core\Render\RendererInterface $renderer
+   *   The renderer service.
+   */
+  public function __construct(
+    TranslationInterface $string_translation,
+    UserPermsService $user_perms_service,
+    EntityTypeManager $entity_type_manager,
+    RendererInterface $renderer
+    ) {
+    $this->stringTranslation = $string_translation;
+    $this->userPermsService = $user_perms_service;
+    $this->entityTypeManager = $entity_type_manager;
+    $this->renderer = $renderer;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function getSubscribedEvents(): array {
+    return [
+      EntityHookEvents::ENTITY_VIEW_ALTER => 'entityViewAlter',
+    ];
+  }
+
+  /**
+   * Alteration to entity view pages.
+   *
+   * @param \Drupal\core_event_dispatcher\Event\Entity\EntityViewAlterEvent $event
+   *   The entity view alter service.
+   */
+  public function entityViewAlter(EntityViewAlterEvent $event):void {
+    $this->appendServiceTermDescriptionToVbaFacility($event);
+  }
+
+  /**
+   * Appends VBA facility service description to title on facility node:view.
+   *
+   * @param \Drupal\core_event_dispatcher\Event\Entity\EntityViewAlterEvent $event
+   *   The entity view alter service.
+   */
+  public function appendServiceTermDescriptionToVbaFacility(EntityViewAlterEvent $event):void {
+    if ($event->getDisplay()->getTargetBundle() === 'vba_facility') {
+      $build = &$event->getBuild();
+      $services = $build['field_vba_services'] ?? [];
+      foreach ($services as $key => $service) {
+        $description = new FormattableMarkup('', []);
+        if (is_numeric($key) && !empty($service['#options']['entity'])) {
+          $service_node = $service['#options']['entity'];
+          $referenced_term_array = $service_node->get('field_service_name_and_descripti')->referencedEntities();
```

Comment:
Going a little hungarian notation here... which I used to love, but a more modern drupallly pattern is to name plurals if unknown quantity and singular when you have one.
```suggestion
          $referenced_terms = $service_node->get('field_service_name_and_descripti')->referencedEntities();
```